evalml/tests/data_checks_tests/test_invalid_targets_data_check.py

```diff
@@ -54,13 +54,6 @@ def test_invalid_target_data_check_nan_error():
     assert invalid_targets_check.validate(X, y=pd.Series([np.nan, np.nan, np.nan])) == {
         "warnings": [],
         "errors": [
-            DataCheckError(
```

Comment:
The slightly different logic catches fully null Unknown columns and changes them to np.nan Double columns both in dataframes and in series now.